.github/workflows/linting.yml

```
@@ -0,0 +1,37 @@
+name: "Linting"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  title:
+  
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: amannn/action-semantic-pull-request@v3.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          requireScope: true
```

```suggestion
          types: |
            fix
            feat
            refactor
            docs
          requireScope: true
```
For this project, we would end up with those as well

---

I agree with the requirement of a scope, however, any scope.
And so, based on the documentation, I am not sure that this is possible.

---

The types from the [specification](https://github.com/commitizen/conventional-commit-types/blob/master/index.json) are automatically allowed. If you want to restrict it, you can define `types`. But then I would also add `chore`.

Yes, that's unfortunately how I understood it, which is why I gave the `types`. But I think that these should be sufficient.